src/config.example.yaml

```
@@ -32,11 +32,36 @@ history:
+  - algorithm: nearestNode
+    props:
+      # Add algorithm-specific properties here
+      maxDistance: 10
+      weightByDistance: true
+
 weighting:
   algorithm: gaussian
   props:
     sigma: 0.10
 
+locators:
```

yamllint

[error] duplication of key "locators" in mapping

(key-duplicates)